cmd/config/config.go

```diff
@@ -3,51 +3,63 @@ package config
 import (
 	"fmt"
 
-	"kaffine-mod/kaffine"
+	"github.com/konveyor/kaffeine/kaffeine"
 
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 )
 
 // support both local and global config
-// global config determined by KAFFINE_GLOBAL_CONFIG env variable.
-// If unset, defaults to ~/.kaffine/config
+// global config determined by kaffeine_GLOBAL_CONFIG env variable.
```

Comment:
nit I think you still want it to be capitalized

Reply:
This was definitely a find-and-replace error, whoops!